tests/integrations/raftstore/test_merge.rs

```diff
@@ -1266,3 +1266,55 @@ fn test_sync_max_ts_after_region_merge() {
     let new_max_ts = cm.max_ts();
     assert!(new_max_ts > max_ts);
 }
+
+#[test]
+fn test_node_base_merge() {
```

Comment:
I have no idea about how to verify the causality here now. Let us have a discussion later.